3-bull-jobs/finished-app/lib/movies/movieApiService.js

```diff
@@ -53,7 +53,7 @@ class MovieAPIService {
 
     const requests = [];
 
-    for (let page = 1; page < 1001; page++) {
+    for (let page = 1; page < 500; page++) {
```

Comment:
Why not do 1000 queue requests?

Reply:
It timedout too often. When it didn't, it took too long to show the purpose of the queues. I reduced it based on practically. Can revert if needed.

Reply:
Enqueue is timing out... weird thing to happen. Gonna try tonight

Reply:
'node-fetch' is timing out and exiting the queue with an exception. We could increase the the timeout number or reduce the page looping number I think.

Reply:
Yeah... is not node fetch really, but maybe us trying the app with same key causing api to be slow... Yeah removing timeout or increasing is the way